src/ui/server/api/keystone/application/index.ts

```diff
@@ -215,6 +216,36 @@ const application = {
         throw new Error('Updating application exporter company');
       }
     },
+    buyer: async (id: string, update: object) => {
+      try {
+        console.info('Updating application buyer');
+
+        const variables = {
+          where: { id },
+          data: update,
+        };
+
+        const response = (await apollo('POST', updateBuyerMutation, variables)) as ApolloResponse;
+
+        if (response.errors) {
+          console.error('GraphQL error updating application buyer', response.errors);
```

Comment:
Can we add a space at the end of the message, and in the `console.error` below, just to be consistent